components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -1162,6 +1162,23 @@ public class MonitoredItemWrapper {
                         }
                     }
 
+                    Task.Run(() => {
```

Comment:
Why this?

Reply:
Alex specifically wanted it done asynchronously

Reply:
@koepalex Thoughts on this?

Reply:
We already have issues for other customer if something isn't executed asynchrounsly. Because it blocks the execution (delivery of telemetry). Each feature that isn't mandatory to wait (like here the convenience feature to detect wrong configurations) should be done as default async and without waiting.

Reply:
Based on experience lookup on 1000 OPC UA nodes, adds around 10 seconds. But if this is pretty fast (< 250ms) in worse case scenario (each event points to unique type). Then it would be fine to execute synchronously.

Reply:
I see. Thanks for the context.